ckanext/qdes_schema/qdes_ckan_dataset.json

```diff
@@ -668,6 +662,10 @@
       "vocabulary_service_name": "format",
       "required": true,
       "form_attrs": {
+        "data-module": "qdes_autocomplete",
```

Comment:
These attributes are already in the preset `qdes_vocab_service_autocomplete`, just need to refactor ` "data-module-title": "title"` into the preset and then you can remove these `form_attrs`